Sources/AccountSDKIOSWeb/Client/Client.swift

```diff
@@ -150,7 +151,14 @@ public class Client: CustomStringConvertible {
                 return
             }
 
-            self.handleAuthenticationResponse(url: url, completion: completion)
+            self.handleAuthenticationResponse(url: url) { result in
+                DispatchQueue.main.async {
+                    if let vc = window?.rootViewController {
+                        vc.dismiss(animated: true, completion: nil)
+                    }
+                }
+                completion(result)
```

Comment:
Will this not affect all types of login? Even the ones outside of Simplified Login